lib/settings/settings.js

```diff
@@ -103,6 +103,12 @@ class Settings {
   get testWorkersEnabled() {
     const {test_workers} = this.settings;
 
+    if (this.argv.parallel === false || this.argv.parallel === 'false' 
+        || this.argv.sequential === true 
+        || isNumber(this.argv.workers) && this.argv.workers === 1) {
+      return false;
+    }
+
     return this.argv.parallel === true || test_workers === true || isObject(test_workers) && test_workers.enabled;
```

Comment:
I think this can be refactored into:
```js
this.argv.sequential ? false : this.argv.parallel ==== true || (test_workers && test_workers.enabled && test_workers.workers > 1)
```

Reply:
I have kept the negatives first. It is supposed to be always enabled, so I have kept the conditions which can disable parallelism together. I think this is more readable.